### PR TITLE
Rebuild v1.22

### PR DESCRIPTION
**Pull request by Zeitsperre**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

The current build of wily on conda-forge seems to be broken. This is a PR to figure out what needs to be fixed.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@tonybaloney This is now a fully functioning package. Feel free to review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,11 +10,9 @@ source:
   sha256: 0b78abeb264d1810a245e563964f1b5d9f380afae75fff032b4499cf300598be
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  entry_points:
-    - wily = wily:main
```
Comment (tonybaloney) [inline recipe/meta.yaml:17]: why is the entry point removed? Won't this mean the `wily` command isn't registered?

Comment (Zeitsperre) [reply recipe/meta.yaml:17]: If the entrypoint is properly configured for PyPI, setting this information here as well will only cause issues. With the current build of the package, I receive the following error:
```
(wily_env) ~ >>> wily                                                                  
Traceback (most recent call last):
  File "/home/me/miniconda3/envs/wily_env/bin/wily", line 6, in <module>
    from wily import main
ImportError: cannot import name 'main' from 'wily' (/home/me/miniconda3/envs/wily_env/lib/python3.9/site-packages/wily/__init__.py)
```

I have added a smoke test below to ensure the CLI is working properly when the entrypoint configuration is removed (and it is). I think no further changes are needed!

Comment (Zeitsperre) [reply recipe/meta.yaml:17]: @tonybaloney Please let me know if you require any clarifications!

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**